setup.cfg

```diff
@@ -1,2 +1,26 @@
 [metadata]
 description-file = README.md
+
+
+[flake8]
+max-line-length = 88
```

Comment:
I know this is very personal but if we change this from the default, can we use 120? like [pytorch](https://github.com/pytorch/pytorch/blob/master/.flake8). I feel it is much easier to read, especially using github code review.

Reply:
Why not :)
Changed